releasenotes/README.md

```diff
@@ -1,148 +1,28 @@
-# Salesforce CLI v50 Release Notes
+# Salesforce CLI v51 Release Notes
 
-Here are the new and changed features in recent updates of Salesforce CLI and the `salesforcedx` plug-in. 
+Here are the new and changed features in recent updates of Salesforce CLI and the `salesforcedx` plug-in.
 
-We publish the `latest` plug-in and CLI on Thursdays. At the same time we also publish the `latest-rc` release candidate plug-in and CLI. The release candidates contain changes that will likely be in the final official version. 
+We publish the `latest` plug-in and CLI on Thursdays. At the same time we also publish the `latest-rc` release candidate plug-in and CLI. The release candidates contain changes that will likely be in the final official version.
 
 Run `sfdx version` to display the version of Salesforce CLI installed on your computer. Run `sfdx plugins --core` to display the version of the installed `salesforcedx` plug-in.
 
 Run `sfdx update` to update both the CLI and the `salesforcedx` plug-in to the latest available version.
 
 If you use [autocomplete](https://developer.salesforce.com/docs/atlas.en-us.sfdx_setup.meta/sfdx_setup/sfdx_dev_cli_autocomplete.htm), run `sfdx autocomplete --refresh-cache` after you update the `salesforcedx` plug-in to ensure that autocomplete works correctly on any new commands.
 
-[Click here for the v49 release notes.](./v49.md)
+[Click here for the v50 release notes.](./v50.md)
 
-## February 11, 2021
+## 51.1.0 (February 18, 2020) - CLI 7.88.3
 
-These changes are in the release candidate plug-in (`salesforcedx@latest-rc`). We plan to include these changes in the next official release. This list isn't final and is subject to change.
+* FIX: The command `force:user:create` properly authenticates to a connected app authenticated with Web Flow login
```

Comment:
Potential edit (because I don't really understand the issue, my suggestion might change the meaning, so reject if so):  "The command `force:user:create` properly authenticates to the connected app associated with the Web Server Flow authentication." 

If that's incorrect, keep what you have but change "Web Flow" to "Web Server Flow" (to match the dev guide terminology) and add a period at the end.